### PR TITLE
Updates python example

### PR DESCRIPTION
**Pull request by seancook**
Updates Python example

* Provides samples that work for Python2 and Python3 
* Adds demo flask app
* Removes dependency on `cryptography` module

### CODE REVIEW

**Comment by seancook** [comment]
@blakewatters @maju6406 @thomas-layer

File: identity-services-samples/python/DJANGO_README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+```
+
+### Configure your Layer app
+
+There are 3 constants that you should set in layer.py, all of which are available in the **Keys** section of the Layer dashboard for your app.
+
+* `PROVIDER_ID` - Provider ID found in the Layer Dashboard under "Authentication"
+* `KEY_ID` - Public key generated and stored in the Layer Dashboard under "Authentication"
+* `RSA_KEY_PATH` - Path to the file containing the private key associated with the public key
+
```
Comment (thomas-layer) [inline identity-services-samples/python/DJANGO_README.md]: The `PROVIDER_ID` and `KEY_ID` are in the Layer Dashboard under "Keys"

File: identity-services-samples/python/FLASK_README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+There are 3 constants that you must be set in layer.py, all of which are
+available in the **Keys** section of the Layer dashboard for your app.
+
+* `PROVIDER_ID` - Provider ID found in the Layer Dashboard under
+"Authentication"
+* `KEY_ID` - Public key generated and stored in the Layer Dashboard under
+"Authentication"
+* `RSA_KEY_PATH` - Path to the file containing the private key associated with
+the public key
+
```
Comment (thomas-layer) [inline identity-services-samples/python/FLASK_README.md]: The PROVIDER_ID and KEY_ID are in the Layer Dashboard under "Keys"

File: identity-services-samples/python/layer.py [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        payload={
+            # String - The Provider ID found in the Layer Dashboard
+            "iss": PROVIDER_ID,
+            # String - Provider's internal ID for the authenticating user
+            "prn": user_id,
+            # Integer - Time of Token Issuance in RFC 3339 seconds
+            "iat": datetime.now(),
+            # Integer - Arbitrary Token Expiration in RFC 3339 seconds
+            "exp": datetime.utcnow() + timedelta(seconds=30),
+            "nce": nonce    # The nonce obtained via the Layer client SDK.
```
Comment (thomas-layer) [inline identity-services-samples/python/layer.py]: 30 seconds is a rather short time for the JWT to expire.

File: identity-services-samples/python/README.md [hunk 4]
```diff
@@ -1,26 +1,56 @@
...
+all of which are available in the **Keys** section of the Layer dashboard for
+your app.
+
+* `PROVIDER_ID` - Provider ID found in the Layer Dashboard under
+"Authentication"
+* `KEY_ID` - Public key generated and stored in the Layer Dashboard under
+"Authentication"
+* `RSA_KEY_PATH` - Path to the file containing the private key associated with
+the public key
+
```
Comment (thomas-layer) [inline identity-services-samples/python/README.md]: The PROVIDER_ID and KEY_ID are in the Layer Dashboard under "Keys"

**Comment by seancook** [comment]
@janin @nilgradisnik - please take look when you have a chance

File: identity-services-samples/python/README.md (hunk 4 above)
Comment (seancook) [inline identity-services-samples/python/README.md]: Thanks. Updated in the latest PR.

File: identity-services-samples/python/layer.py (hunk 3 above)
Comment (seancook) [inline identity-services-samples/python/layer.py]: Thanks, updated to 2 minutes (120 seconds) in the latest PR. The spec says that "a few minutes" is the high water mark for expiration.

File: identity-services-samples/python/layer.py [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+PROVIDER_ID = ''
+KEY_ID = ''
+RSA_KEY_PATH = "/path_to/layer.pem"
+
+if not (PROVIDER_ID and KEY_ID and RSA_KEY_PATH):
+    raise Exception("You must provide PROVIDER_ID, KEY_ID, and "
+                    "RSA_KEY_PATH in %s" % __file__)
+
+
+def read_rsa_private_key(file_path=RSA_KEY_PATH):
```
Comment (nilgradisnik) [inline identity-services-samples/python/layer.py]: Minor thing, I'd make this function "private" which in python terms is something like `__read_rsa_private_key`.

Comment (seancook) [inline identity-services-samples/python/layer.py]: Thanks, updated it to "_read_rsa_private_key" - the double underscore prefix is only for python internals

Comment (nilgradisnik) [inline identity-services-samples/python/layer.py]: Ok that works. I was always confused about the double vs single underscore prefix in python, some people claim that single is protected and double is private. :smile:

**Pull request merged**